### ISSUE TITLE
Collaborative Video Object Segmentation by Foreground-Background Integration 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#12]
# TL;DR
In semi-supervised video segmentation (VOS), this study aimed to improve accuracy by incorporating background information as well, rather than using only the foreground information as in previous studies. Good quantitative and qualitative results in DAVIS 2016 etc .
![image](https://user-images.githubusercontent.com/24524018/89390799-38dc5a80-d742-11ea-8d20-0ed81439ff2a.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2003.08333

# Submission Dates(yyyy/mm/dd)
2020/05/18

# Authors and institutions
Zongxin Yang, Yunchao Wei, and Yi Yang
-Baidu Research - ReLER, Centre for Artificial Intelligence, University of Technology Sydney

# Methods
The key is that the background as well as the foreground is embedded.
![image](https://user-images.githubusercontent.com/24524018/89390739-1c402280-d742-11ea-8a53-9651d0a64e9b.png)

# Results
DAVIS 2016 and others, qualitatively and quantitatively better than previous studies.
![image](https://user-images.githubusercontent.com/24524018/89390759-2530f400-d742-11ea-8289-e0d5d00123b7.png)
![image](https://user-images.githubusercontent.com/24524018/89390767-295d1180-d742-11ea-85cd-efbdcf9c4176.png)

# Comments
